libs/plugins/stream-client/src/lib/core/state/clean-dangling-tasks-output-mappings.ts

```diff
@@ -65,3 +62,7 @@ function cleanTasks(
   }
   return items;
 }
+
+function normalizeOutputs(outputs) {
+  return outputs.map(output => `$${ROOT_TYPES.PIPELINE}.${output}`);
```

Comment:
outputs should be `pipeline.name` and not `$pipeline.name` (dollar sign). Problem is that then this won't work because the outputs are currently generated incorrectly in the mapper, that issue is tracked by #1170 . Maybe we can include that fix in this PR?